ClimsoftVer4/ClimsoftVer4/formDaily1.vb

```diff
@@ -1,3 +1,19 @@
-﻿Public Class formDaily1
+﻿
```

Comment:
Another very minor comment... could you remove the empty line at the top of the file here so that it is the same as all the others?